src/services/completions.ts

```diff
@@ -1688,7 +1688,16 @@ namespace ts.Completions {
 
             if (contextToken.kind === SyntaxKind.GreaterThanToken && contextToken.parent) {
                 if (contextToken.parent.kind === SyntaxKind.JsxOpeningElement) {
-                    return true;
+                    // Two possibilities:
+                    //   1. <div>/**/

```

Comment:
Is `/**/` an existing convention for cursor location?  I missed it until I got to the explanatory text (I'd usually expect a `|` somewhere).

Reply:
Yeah, because it’s working fourslash syntax, and it doesn’t mess up syntax highlighting in a markdown codeblock.